### ISSUE TITLE
Support for more ANSI color codes

### ISSUE THREAD

**Issue by frun36** [shohamp/culour#5]
Hi!
Why aren't more ANSI color codes supported? In particular, I'm missing the remaining 3 colors from the most typical 8 colors (see https://gist.github.com/fnky/458719343aabd01cfb17a3a4f7296797#8-16-colors).

```python
class TerminalColors(object):
    MAGENTA = '[95'
    BLUE = '[94'
    GREEN = '[92'
    YELLOW = '[93'
    RED = '[91'
    END = '[0'
```
Misses `CYAN`, `BLACK`, `WHITE`.

**Comment by spellr** [comment]
Hey, I'm happy to know someone is using this package.
I'm happy to accept pull requests

**Comment by Hierosme** [comment]
Hello,

I just ping...

The approch of culour is static because NCurses use same default color, that is a perfect match with NCurses.
Clearly not culour fault

If we speach about more of default NCurses color, then a static approch require 256 colors for forground and 256 colors for background (with special cases like for background ALPHA (must be color pair 0)).
Then can can generate (256 * 256) colors pair.

Technical info
Normaly ANSII color ar a 7bits color, the static approch can easilly be translate to logic 7bits, and match with default couler approch., 

NCurses have a function for speack about RGB where (R: range(0, 1000), G: range(0, 1000), B: range(0, 1000)), then can back to a 255 max.

The 256 colors type is a xterm technology, that is a 8bits logic and permit a True  RGB approch.

Why 7Bits or 8Bits ? : On 7Bits a octet is resever for color call blink (use by the cursor prompt),  on 8Bits approch  the blink color do not exist any more, then you can use that octet for better color range. Then 256 colors.

I have to devellop a 256 colors methods for a TUI project to we want it here ?

**Comment by spellr** [comment]
@Hierosme I'm not sure I understand your comment. Let's move it to a new issue, as this issue was resolved

**Comment by spellr** [comment]
resolved with #6
